### PR TITLE
Added a note

### PR DESCRIPTION
**Pull request by joinimran**
Added a note if users are not able to connect using RDP 6.0 then follow the steps mentioned in a support article. 

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/4616

### CODE REVIEW

**Comment by joinimran** [comment]
@JohanFreelancer9 Please do a copy review.

File: windows/client-management/connect-to-remote-aadj-pc.md [hunk 1]
```diff
@@ -55,6 +55,8 @@ Ensure [Remote Credential Guard](/windows/access-protection/remote-credential-gu
   >[!TIP]
   >When you connect to the remote PC, enter your account name in this format: `AzureAD UPN`. The local PC must either be domain-joined or Azure AD-joined. The local PC and remote PC must be in the same Azure AD tenant.
 
+> [!Note]
+> If you cannot connect using Remote Desktop Connection 6.0 then turn off new features of RDP 6.0 and revert back to RDP 5.0 by tweaking a few changes in the RDP file. See the details in the support [article](https://support.microsoft.com/en-ph/help/941641/remote-desktop-connection-6-0-prompts-you-for-credentials-before-you-e).
```
Comment (illfated) [inline windows/client-management/connect-to-remote-aadj-pc.md]: minor details (sentence clause delimiter (comma) and link text inclusion)
```suggestion
> If you cannot connect using Remote Desktop Connection 6.0, then turn off new features of RDP 6.0 and revert back to RDP 5.0 by tweaking a few changes in the RDP file. See the details in the [support article](https://support.microsoft.com/en-ph/help/941641/remote-desktop-connection-6-0-prompts-you-for-credentials-before-you-e).
```

**Comment by illfated** [comment]
Thank you for using the recommended MarkDown quote compatibility space in your Note. Appreciated.

**Comment by jvsam** [comment]
@JohanFreelancer9 for copy review. Thank you!

File: windows/client-management/connect-to-remote-aadj-pc.md [hunk 2]
```diff
@@ -55,6 +55,8 @@ Ensure [Remote Credential Guard](/windows/access-protection/remote-credential-gu
   >[!TIP]
```
Comment (JohanFreelancer9) [inline windows/client-management/connect-to-remote-aadj-pc.md]:
```suggestion
  > [!TIP]
```

File: windows/client-management/connect-to-remote-aadj-pc.md [hunk 3]
```diff
@@ -55,6 +55,8 @@ Ensure [Remote Credential Guard](/windows/access-protection/remote-credential-gu
...
   >When you connect to the remote PC, enter your account name in this format: `AzureAD UPN`. The local PC must either be domain-joined or Azure AD-joined. The local PC and remote PC must be in the same Azure AD tenant.
```
Comment (JohanFreelancer9) [inline windows/client-management/connect-to-remote-aadj-pc.md]:
```suggestion
  > When you connect to the remote PC, enter your account name in this format: `AzureAD UPN`. The local PC must either be domain-joined or Azure AD-joined. The local PC and remote PC must be in the same Azure AD tenant.
```

File: windows/client-management/connect-to-remote-aadj-pc.md (hunk 1 above)
Comment (JohanFreelancer9) [inline windows/client-management/connect-to-remote-aadj-pc.md]:
```suggestion
> If you cannot connect using Remote Desktop Connection 6.0, turn off new features of RDP 6.0 and revert back to RDP 5.0 by tweaking a few changes in the RDP file. For more details, see the [support article](https://support.microsoft.com/en-ph/help/941641/remote-desktop-connection-6-0-prompts-you-for-credentials-before-you-e).
```

**Comment by joinimran** [comment]
Ready to go live

**Comment by jvsam** [comment]
Hello @Dansimp, this pull request is ready for your review and approval. Thank you.

**Comment by jvsam** [comment]
Hi @Dansimp, kindly review this PR when you have time. Thank you.

@officedocsbot assign @Dansimp

**Pull request merged**